source/Calamari/Kubernetes/Commands/KubernetesApplyRawYamlCommand.cs

```diff
@@ -58,9 +61,11 @@ public override int Execute(string[] commandLineArguments)
 
         protected override async Task<bool> ExecuteCommand(RunningDeployment runningDeployment)
         {
-            var gatherAndApplyRawYamlExecutor =
-                new GatherAndApplyRawYamlExecutor(log, fileSystem, kubectl);
-
+            if (!variables.GetFlag(SpecialVariables.ResourceStatusCheck))
```

Comment:
We do not check `DeploymentStyle` and `DeploymentWait` here, although we do this in the [script wrapper](https://github.com/OctopusDeploy/Calamari/blob/1f6e2a6b523eb0a7259d7f47d231d54cbda74636/source/Calamari/Kubernetes/ResourceStatus/ResourceStatusReportWrapper.cs#L38-L44). This is because these settings are only for Deploy-a-container step. We don't have these settings for RAW YAML.

Reply:
Nice and simple. Good work!